ScoreSaber/UI/Leaderboard/ScoreSaberLeaderboardViewController.cs

```diff
@@ -712,28 +706,6 @@ public void OnLeaderboardSet(BeatmapKey beatmapKey) {
             } catch(Exception ex) { Plugin.Log.Error(ex.Message); }
         }
 
-        internal static class BeatmapDataCache {
-            internal static Dictionary<string, Sprite> cachedSprites = new Dictionary<string, Sprite>();
-            private static int MaxSpriteCacheSize = 150;
-            internal static Queue<string> spriteCacheQueue = new Queue<string>();
-            internal static void MaintainSpriteCache() {
-                while (cachedSprites.Count > MaxSpriteCacheSize) {
-                    string oldestUrl = spriteCacheQueue.Dequeue();
-                    cachedSprites.Remove(oldestUrl);
-                    
-                }
-            }
-
-            internal static void AddSpriteToCache(string url, Sprite sprite) {
-                if (cachedSprites.ContainsKey(url)) {
-                    return;
-                }
-                cachedSprites.Add(url, sprite);
-                spriteCacheQueue.Enqueue(url);
-            }
-        }
-
-
         // probably a better place to put this
         public class CellClicker : MonoBehaviour, IPointerClickHandler, IPointerEnterHandler, IPointerExitHandler {
```

Comment:
The comment about moving the separate classes to different files was meant to apply to all the extra classes in here, not just the Cache.

Reply:
fixed